### PR TITLE
Adds tablesaws

### PR DESCRIPTION
**Pull request by Carbonhell**
Adds tablesaws. They are like canvas, but they only allow to cut out pixels.
1) You can make a wooden plate by using a wooden plank on the table saw.
2)You can insert said plate to make the tablesaw show a zoomed in image which can be:
    a) clicked to delete the pixel you're aiming
    b) hold alt and move the mouse around to delete multiple pixels
As weird as it may sound,this requires a bit of graphical process so if you try to delete shit too fast it'll lag and the image may disappear for a few seconds, to then appear again with the proper changes made.

Other features:
Grab aggressively and put someone on the table saw, causing them to get 50 brute damage and lose the limb being targeted. Due to the huge damage i didn't make it knock the user down too but the damage will be enormous,like meat spike. To table them you need to wait 12 seconds,like meatspike.

### CODE REVIEW

**Comment by doobiedoo23** [comment]
Sure

File: _maps/map_files/TgStation/tgstation.2.1.3.dmm [hunk 1]
```diff
@@ -2542,7 +2542,7 @@
 "aWT" = (/obj/structure/disposalpipe/segment,/obj/machinery/atmospherics/pipe/manifold/scrubbers/hidden{tag = "icon-manifold-r-f (EAST)"; dir = 4},/turf/simulated/floor/plasteel{icon_state = "hydrofloor"},/area/hydroponics)
 "aWU" = (/obj/structure/closet/wardrobe/botanist,/obj/machinery/atmospherics/pipe/simple/supply/hidden,/obj/machinery/light_switch{pixel_y = 28},/turf/simulated/floor/plasteel{icon_state = "hydrofloor"},/area/hydroponics)
 "aWV" = (/obj/structure/closet/secure_closet/hydroponics,/turf/simulated/floor/plasteel{icon_state = "hydrofloor"},/area/hydroponics)
-"aWW" = (/obj/machinery/alarm{pixel_y = 24},/obj/machinery/camera{c_tag = "Hydroponics Storage"},/obj/machinery/light/small{dir = 1},/turf/simulated/floor/plasteel{icon_state = "hydrofloor"},/area/hydroponics)
+"aWW" = (/obj/machinery/alarm{pixel_y = 24},/obj/machinery/camera{c_tag = "Hydroponics Storage"},/obj/machinery/light/small{dir = 1},/obj/structure/tablesaw,/obj/item/weapon/paper/tablesaw,/turf/simulated/floor/plasteel{icon_state = "hydrofloor"},/area/hydroponics)
```
Comment (JohnGinnane) [inline _maps/map_files/TgStation/tgstation.2.1.3.dmm:2545]: Could we possibly add this table saw to the biogenerator?

**Comment by Carbonhell** [comment]
Currently it cannot be made nor moved, because i've placed it in such a place where it doesn't clutter the visual. I could do that but if you moved it and put it in the center of the hallway it'd clutter your visual a bit,through i can easily do it

**Comment by JohnGinnane** [comment]
Can you post a screenshot of what it looks like when you're sawing an item?

**Comment by Carbonhell** [comment]
http://imgur.com/elqqK1U

**Comment by JohnGinnane** [comment]
Thanks, and code looks good. What causes the lag when you move too fast?

**Comment by Carbonhell** [comment]
Image rendering, just that. Trying to edit the icon directly breaks it,so you need to make an image holder,edit it, and then set the plate icon to it and update the zoom icon. For every edited pixel. But it's bearable,so. Btw only one headmin ok left, so please whoever's gonna ok this merge it too

**Comment by myontasm** [comment]
Sure, why not.

**Pull request merged**